Makefile

```diff
@@ -10,11 +10,17 @@ TGTS = graphql jsonschema docs shex owl csv graphql python
 #GEN_OPTS = --no-mergeimports
 GEN_OPTS =
 
-all: gen stage
+# We run all commands in the Pipenv.
+RUN = pipenv run
+
+all: pipenv-install gen stage
+pipenv-install:
+	pipenv install
 gen: $(patsubst %,gen-%,$(TGTS))
```

Comment:
Should `gen` be `PHONY`? (and `pipenv-install` depending on feelings on previous comment))

Reply:
Yup! Fixed in 37af910cf42cf0.